src/pages/user/Profile.js

```diff
@@ -261,234 +259,17 @@ export class Profile extends Component {
                 </Button>
               )
             )}
-            {editting && this.profileEditor()}
+            {editing && this.profileEditor()}
```

Comment:
This could be a getter if it doesn't accept any arguments

Reply:
Updated